src/extension/features/budget/display-total-overspent/settings.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  name: 'DisplayTotalOverspent',
+  type: 'checkbox',
+  default: false,
+  section: 'budget',
+  title: 'Display Total Overspent',
```

Comment:
can we rename this to `Show Total Overspent` to be more in line with how similar features are named?